GliaWidgets/Sources/ViewController/Survey/SurveyViewController.View.swift

```diff
@@ -189,7 +189,7 @@ extension NSAttributedString {
         if isRequired {
             mutableString.append(
                 .init(
-                    string: L10n.Survey.Question.Title.asterisk,
+                    string: "*",
```

Comment:
would we like to prevent customization for required fields marker? I mean if integrator would like to change asterisk symbol to "required" label

Reply:
What could be done is to expose it through the theme. But we decided with Karl to remove it because an asterisk on its own is not localizable, and also this one had some displaying logic because it wasn't only the asterisk, but an asterisk with a space. Would exposing it through the theme be acceptable?

Reply:
I'm not a fun of moving this kind of customization to theme (i suppose to have only ui in the theme structure), but yes, perhaps in future we can create some separate structure for this kind of customization.